versioned_docs/version-0.4.0/kubeslice-cli/troubleshooting-guide.mdx

```diff
@@ -36,3 +38,11 @@ error: the server doesn't have a resource type "sliceconfigs"
 - Ensure you are on the controller cluster to run the commands: `kubectx -c`.
 - Export the configuration file using this command: `export KUBECONFIG=kubeslice/<path-to-the-kubeconfig-file>`.
 
+
+### Getting an Unverified Developer Error message on MAC
```

Comment:
I think we need to write MAC as `MacOS` to be more clear. 

I remember @priyank-upadhyay  asking us to refer to MacOS.